doc/magic_cards_notes.md

```diff
@@ -1008,7 +1008,62 @@ If the card is an Ultimate Magic Card, it returns 30 bytes.
 ### Magic commands
 ^[Top](#top) ^^[Gen4](#g4top)
 
-Special commands summary:
+script run hf_mf_ultimatecard.lua -h
```

Comment:
Add a short text explaning,   that programming the tag can be done with raw commands or script.
and that we are giving examples how to do both.

Reply:
Done